integrations/openai/src/index.ts

```diff
@@ -59,23 +64,141 @@ const models: Record<ModelId, interfaces.llm.ModelDetails> = {
   },
 }
 
+const imageModels: Record<ImageModelId, interfaces.textToImage.ImageModelDetails> = {
+  'dall-e-3-standard-1024': {
```

Comment:
**note**: The pricing for DALL-E is based on the model version (v2 vs v3), the quality (standard vs hd), and the max size (1024px, 1792px, etc.). This approach to structuring/pricing the models will allow us to easily adapt to different pricing schemas also, such as Stable Diffusion's.